docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -540,7 +540,7 @@ in the metastore. As a result, any Databricks engine can write to the table::
 The Delta Lake connector also supports creating tables using the :doc:`CREATE
 TABLE AS </sql/create-table-as>` syntax.
 
-There are three table properties available for use in table creation.
+There are four table properties available for use in table creation.
```

Comment:
`There are four table properties available for use` -> `The following properties are available for use` 

The actual count of properties available is not necessary to be known by the end user.

Reply:
I merely updated the number that was there already but sure I can rephrase